prod/docker-compose.yml

```diff
@@ -1,8 +1,15 @@
 version: '2'
 services:
-    geodocker-pbase:
-        build: ./base
+    geodocker-ubuntu-pbase:
+        build:
+          context: ./base
+          dockerfile: ubuntu.dockerfile
         image: daunnc/geodocker-pbase:latest
+    geodocker-centos-pbase:
+        build:
+          context: ./base
+          dockerfile: centos.dockerfile
+        image: moradology/geobase:latest
```

Comment:
Would be great if images names would be consistent like `daunnc/geodocker-ubuntu-base:latest` and `moradology/geodocker-centos-base:latest`